modules/apps/headless/headless-form/headless-form-api/src/main/java/com/liferay/headless/form/dto/v1_0/util/StructureUtil.java

```diff
@@ -154,6 +154,8 @@ private static FormField _toFormField(
 		LocalizedValue labelLocalizedValue = ddmFormField.getLabel();
 		LocalizedValue predefinedLocalizedValue =
 			ddmFormField.getPredefinedValue();
+		LocalizedValue placeholderLocalizedValue =
+			ddmFormField.getPlaceholder();
```

Comment:
@joaovictoralvesmartins, can you please add this lines before `predefinedLocalizedValue` declaration to keep alphabetical sort?

Thanks.

Reply:
Done!